### PR TITLE
Add note at the Reaction CLI doc on how to install the CLI

### PR DESCRIPTION
**Pull request by mpaktiti**
Issue: Resolves unreported issue
Impact: **minor**  
Type: **docs**

## Issue
The [Reaction CLI](https://docs.reactioncommerce.com/docs/next/reaction-cli) doc lists various commands but it doesn't mention installing it. I found this information in the [Reaction Platform](https://docs.reactioncommerce.com/docs/next/deploying-on-reaction-platform) doc and I think it would be useful to have it also here. I also found a typo in the second doc, which I'm fixing in the same PR. 

## Solution & Screenshots
Added a sentence and a code snippet telling people how to install the CLI. Also, fixed a typo in another doc. 

## Breaking changes
None.

## Testing
1. Check that the new sentence + code snippet has been added at the [Reaction CLI](https://docs.reactioncommerce.com/docs/next/reaction-cli) doc
2. Check that the "Beforey" typo has been removed from [Reaction Platform](https://docs.reactioncommerce.com/docs/next/deploying-on-reaction-platform) doc

### CODE REVIEW

**Comment by machikoyasuda** [comment]
Hi @mpaktiti - Woot for this first PR 🎉 

1. The CLI page: I created this issue in response to this PR, https://github.com/reactioncommerce/reaction-docs/issues/816, because one of the recent 2.0 Reaction release candidates removed the core app's dependency on https://github.com/reactioncommerce/reaction-cli from 2.0 and beyond. Changelog note here: https://github.com/reactioncommerce/reaction/blob/master/CHANGELOG.md#performance-tweaks PR here: https://github.com/reactioncommerce/reaction/pull/4992

As you can see from this repo: https://github.com/reactioncommerce/reaction-cli - this repo is now archived and unable for people to update. But thanks to your issue here I realized that the NPM module for the CLI is still not updated with this latest README change: https://www.npmjs.com/package/reaction-cli - I've alerted @spencern that this needs to be updated on the NPM level.

2. The Reaction Platform docs: I'm afraid we're also probably going to delete/remove that Reaction Platform deployment doc too: https://docs.reactioncommerce.com/docs/next/deploying-on-reaction-platform -- it's referring to this service "Launchdock" that we have removed/dropped entirely. This "Launchdock" was also referred to as the "Reaction Platform" but is different from this: https://github.com/reactioncommerce/reaction-platform -- It's so confusing! There's an outstanding issue we need to resolve soon regarding that doc: https://github.com/reactioncommerce/reaction-docs/issues/710

If you want to take this one: https://github.com/reactioncommerce/reaction-docs/issues/816 - you can take this issue and convert this PR to resolve this. (But you don't have to, no worries).

**Comment by mpaktiti** [comment]
Thanks for the detailed explanation @machikoyasuda! 

I'll take over https://github.com/reactioncommerce/reaction-docs/issues/816 and update this PR.

**Comment by machikoyasuda** [comment]
Checking locally - looks good!
Screen Shot 2019-07-01 at 9 48 23 AM
Screen Shot 2019-07-01 at 9 48 28 AM

Note: new changes go to the `/next/` version of the URL

**Comment by mpaktiti** [comment]
@machikoyasuda Ready for review!

Fixes https://github.com/reactioncommerce/reaction-docs/issues/816

File: public-docs/reaction-cli.md [hunk 1]
```diff
@@ -3,81 +3,49 @@ id: reaction-cli
 title: Reaction CLI
 ---
 
-Reaction ships with all the standard Meteor commands, as documented in the [Meteor Guides](https://docs.meteor.com/commandline.html), along with those from Reaction's own command line interface (CLI). Here are some handy commands to use while developing on Reaction:
+**Note:** As of Reaction 2.0 the CLI is deprecated and unsupported. You can use the docker commands listed in this doc, to achieve the same results.
```
Comment (machikoyasuda) [inline public-docs/reaction-cli.md]:
```suggestion
> **Note:** As of Reaction 2.0 the CLI is deprecated and unsupported. You can use the Docker commands listed in this doc, to achieve the same results.
```

**Comment by spencern** [comment]
Following up here - I've added a deprecation warning to the `reaction-cli` package on npm.
https://www.npmjs.com/package/reaction-cli

**Pull request merged**